kong/pdk/service/response.lua

```diff
@@ -285,10 +285,11 @@ local function new(pdk, major_version)
   -- -- or `access` phase prior calling this function.
   --
   -- local body = kong.service.response.get_raw_body()
-  function response.get_raw_body()
+  function response.get_raw_body(buffered_proxying)
```

Comment:
just a minor refinement so that if we could reduce the reads of `ngx.ctx.buffered_proxying` by passing it by param